css/index.css

```diff
@@ -389,6 +389,11 @@ input[type='file']::before {
   grid-auto-columns: 1fr;
 }
 
+.fade-in-dark,
+.fade-in-light {
+  transition: 0.5s linear;
```

Comment:
We can make the transition property `0.3s ease`, this way the transition will be faster and fade will look smooth.

Reply:
Sure, that looks better ! Will do the changes.